src/encoded/tests/test_item_utils_sample.py

```diff
@@ -0,0 +1,120 @@
+from typing import Any, Dict, List
+
+import pytest
+from webtest import TestApp
+
+from .utils import get_search
+from ..item_utils import item
+from ..item_utils.sample import (
+    get_sample_descriptions,
+    get_sample_names,
+    get_studies,
+)
+from ..item_utils.utils import RequestHandler
+
+
+@pytest.mark.workbook
```

Comment:
FYI if the entire file is workbook tests you can put 

```pytestmark = [pytest.mark.workbook]```

at the top of the file and it will automatically mark all tests in the file as workbook tests.